cirq-core/cirq/experiments/qubit_characterizations.py

```diff
@@ -230,6 +230,70 @@ def single_qubit_randomized_benchmarking(
     return RandomizedBenchMarkResult(num_clifford_range, gnd_probs)
 
 
+def parallel_single_qubit_randomized_benchmarking(
+    sampler: 'cirq.Sampler',
+    use_xy_basis: bool = True,
+    *,
+    qubits: Optional[Iterator['cirq.GridQubit']] = None,
+    num_clifford_range: Sequence[int] = [5, 18, 70, 265, 1000],
+    num_circuits: int = 10,
+    repetitions: int = 600,
+) -> dict:
+    """Clifford-based randomized benchmarking (RB) single qubits in parallel.
+
+    This is the same as `single_qubit_randomized_benchmarking` except on all
+    of the specified qubits in parallel, i.e. with the individual randomized
+    benchmarking circuits zipped together.
+
+    Args:
+        sampler: The quantum engine or simulator to run the circuits.
+        use_xy_basis: Determines if the Clifford gates are built with x and y
+            rotations (True) or x and z rotations (False).
+        qubits: The qubits to benchmark. If None, benchmark all of the qubits.
+        num_clifford_range: The different numbers of Cliffords in the RB study.
+        num_circuits: The number of random circuits generated for each
+            number of Cliffords.
+        repetitions: The number of repetitions of each circuit.
+
+    Returns:
+        A dictionary from qubits to RandomizedBenchMarkResult objects.
+    """
+
+    if qubits is None:  # pragma: no cover
+        try:
+            device = sampler.processor.get_device()  # type: ignore
+        except:
+            device = sampler.device  # type: ignore
+        qubits = device.metadata.qubit_set
+
+    cliffords = _single_qubit_cliffords()
+    c1 = cliffords.c1_in_xy if use_xy_basis else cliffords.c1_in_xz
+    cfd_mats = np.array([_gate_seq_to_mats(gates) for gates in c1])
+
+    # create circuits
+    circuits_all = []
+    for num_cfds in num_clifford_range:
+        for _ in range(num_circuits):
+            circuits_all.append(_create_parallel_rb_circuit(qubits, num_cfds, c1, cfd_mats))
+
+    # run circuits
+    results_all = sampler.run_batch(circuits_all, repetitions=repetitions)
+    gnd_probs: dict = {q: [] for q in qubits}
+    idx = 0
+    for num_cfds in num_clifford_range:
+        excited_probs_l: dict = {q: [] for q in qubits}
+        for _ in range(num_circuits):
+            results = results_all[idx][0]
+            for qubit in qubits:
+                excited_probs_l[qubit].append(
+                    np.mean(results.measurements[f"q{qubit.row}_{qubit.col}"])
```

Comment:
As noted below, can omit the key func when calling `measure_each` then use `str(q)` here to look up the results.
```suggestion
                    np.mean(results.measurements[str(q)])
```